app/src/main/java/bean/pwr/imskamieskiego/NavigationWindow/NavWindowFragment.java

```diff
@@ -56,9 +76,44 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
     public void onPause() {
         super.onPause();
         navWindowListener.onBack();
+    }
 
+    @Override
+    public void onResume() {
+        super.onResume();
+        resumeCounter++;
+    }
+
+    public void sendChangeFloorButtonCoords(){
+        ViewTreeObserver viewTreeObserver = navigationBar.getViewTreeObserver();
+        viewTreeObserver.addOnGlobalLayoutListener(new ViewTreeObserver.OnGlobalLayoutListener() {
+            @Override
+            public void onGlobalLayout() {
+                navigationBar.getViewTreeObserver().removeOnGlobalLayoutListener(this);
+                navWindowListener.setChangeFloorButtonCoords(navigationBar.getHeight(), resumeCounter);
+            }
+        });
     }
 
+    public View.OnTouchListener checkOnTouchCoords() {
```

Comment:
Cała ta metoda jest tylko do Twoich testów, tak? To weź albo to usuń, albo jakoś oznacz, że to jest do późniejszego podmienienia czy cokolwiek.